src/client/components/resize-handle/resize-handle.tsx

```diff
@@ -72,56 +60,63 @@ export class ResizeHandle extends React.Component<ResizeHandleProps, ResizeHandl
     event.preventDefault();
   }
 
-  getValueFromX(x: number): number {
-    if (this.props.side !== "right") {
-      return this.constrainValue(x);
-    }
-
-    return this.constrainValue(window.innerWidth - x);
-  }
+  onGlobalMouseUp = () => {
+    this.setState({
+      dragging: false
+    });
+    window.removeEventListener("mouseup", this.onGlobalMouseUp);
+    window.removeEventListener("mousemove", this.onGlobalMouseMove);
 
-  constrainValue(value: number): number {
-    return clamp(value, this.props.min, this.props.max);
+    if (isFunction(this.props.onResizeEnd)) {
+      this.props.onResizeEnd(this.state.currentValue);
+    }
   }
 
-  onGlobalMouseMove(event: MouseEvent) {
+  onGlobalMouseMove = (event: MouseEvent) => {
     const { anchor } = this.state;
+    const currentValue = this.constrainValue(this.getCoordinate(event)) - anchor;
+    this.setState({ currentValue });
+    if (!!this.props.onResize) this.props.onResize(currentValue);
```

Comment:
Why here you are performing `!!this.props.onResize` check, whereas in other place you check `isFunction(this.props.onResizeEnd)`?